components/inputs/input-date-time.js

```diff
@@ -130,7 +177,27 @@ class InputDateTime extends SkeletonMixin(FormElementMixin(LocalizeCoreElement(R
 			dateStyle.paddingRight = '0.3rem';
 		}
 
+		const parsedValue = this.value ? (this.localized ? this.value : getLocalDateTimeFromUTCDateTime(this.value)) : '';
 		const tooltip = (this.validationError && !this._dropdownOpened && this.childErrors.size === 0) ? html`<d2l-tooltip align="start" announced for="${this._inputId}" state="error">${this.validationError}</d2l-tooltip>` : null;
+		const inputTime = !timeHidden ? html`<d2l-input-time
```

Comment:
I switched to only rendering time when there is a value/date selected. Should this only happen on the first render and then after that it's just hidden if the user clears the date?

Reply:
I think this is fine the way you have it now... it adds a small rendering expense but only when a date is added/cleared.